modules/serverless-installing-cli-linux.adoc

```diff
@@ -6,19 +6,31 @@
 [id="installing-cli-linux_{context}"]
 = Installing the Knative CLI for Linux
 
-For Linux distributions, you can download the `kn` CLI directly as a `tar.gz` archive.
+If you are using a Linux distribution that does not have RPM or another package manager installed, you can install the Knative (`kn`) CLI as a binary file. To do this, you must download and unpack a `tar.gz` archive and add the binary to a directory on your `PATH`.
 
-// no prereqs?
+.Prerequisites
+
+* If you are not using RHEL or Fedora, ensure that *libc* is installed in a directory on your library path.
++
+[IMPORTANT]
```

Comment:
In https://deploy-preview-44656--osdocs.netlify.app/openshift-enterprise/latest/serverless/cli_tools/installing-kn.html#installing-cli-web-console_installing-kn , the same warning is just a "Note", while here we're making it `Important` , I think we should make these consistent (either way)

Reply:
updated so that they're both formatted as important and included in the prereqs

Reply:
There are two more of those, 

https://deploy-preview-44656--osdocs.netlify.app/openshift-enterprise/latest/serverless/cli_tools/installing-kn.html#installing-cli-linux-ibm-power-tarball_installing-kn

and

https://deploy-preview-44656--osdocs.netlify.app/openshift-enterprise/latest/serverless/cli_tools/installing-kn.html#installing-cli-linux-ibm-z-tarball_installing-kn